### PR TITLE
Add launch.json to enable debugger

### PR DESCRIPTION
**Pull request by vladkluev**
Sometimes it's nice to have a full GUI debugger. This allows one to run a unit test and attach a debugger to it in vscode

image

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (fpacifici) [inline .vscode/launch.json:1]: Do you actually need this? I do regularly attach vscode debugger to my unit tests without having a launch.json file.

File: .vscode/settings.json [hunk 2]
```diff
@@ -49,7 +49,12 @@
   "python.pythonPath": ".venv/bin/python",
   // test discovery is sluggish and the UI around running
   // tests is often in your way and misclicked
+  // set this to true if you want to be able to click
+  // to debug a test
   "python.testing.pytestEnabled": false,
```
Comment (fpacifici) [inline .vscode/settings.json:54]: Don't you need to set this to `true` ?

File: .vscode/launch.json (hunk 1 above)
Comment (vladkluev) [reply .vscode/launch.json:1]: vscode always tells I need one 🤔

File: .vscode/settings.json (hunk 2 above)
Comment (vladkluev) [reply .vscode/settings.json:54]: Yes but according to the comment above it was set to false for a reason (i.e. query performance) so I didn't set it to true by default

File: .vscode/launch.json (hunk 1 above)
Comment (fpacifici) [reply .vscode/launch.json:1]: interesting. Well, won't really hurt

File: .vscode/settings.json (hunk 2 above)
Comment (fpacifici) [reply .vscode/settings.json:54]: right, but how do you run test to attach the debug to without ?

**Pull request closed without merging**